UPGRADES.md

```diff
@@ -0,0 +1,94 @@
+	- *If using an existing network license manager*:<ol style="list-style-type:lower-roman"><li>From the EC2 instance for your network license manager, locate the public IPv4 DNS and click the corresponding URL. This takes you to the Network License Manager for MATLAB Dashboard login page.</li><li>Log in with the previously configured username and password.</li><li>Navigate to the [reference architecture documentation](https://github.com/mathworks-ref-arch/matlab-production-server-on-aws#deploy-reference-architecture-for-your-release) for the MATLAB release of your new deployment. Follow the additional configuration steps in the “How to use an existing license server in an existing VPC?” subsection, under “How do I use an existing VPC to deploy MATLAB Production Server?” in the FAQ section. (For R2023a and later, follow the steps under "Use Existing License Server in Existing VPC," under "Additional Information.")</li></ol>
+2. Follow the instructions in the dashboard to retrieve and upload the license file. Make sure to choose the release and operating system that match your new deployment. 
+3. (Optional) Deactivate the previous license in the License Center. 
+
+### Step 3. Migrate Applications to New Deployment
+**Note**: These instructions are for deployed applications compiled using one of the six most recent versions of MATLAB Runtime. For all other applications, refer to [Deployed Applications](UPGRADES.md#deployed-applications).
+
+1. Navigate to the S3 bucket containing the applications for your existing deployment. This can be found in the **Outputs** tab for your existing deployment's stack, with the key ```MATLABProductionServerApplicationsBucket```. 
+2. Select all code archives (``.ctf`` files) in the **auto_deploy** folder, then select **Actions > Move**. 
+3. Set the destination as the **auto_deploy** folder in the S3 bucket for your new deployment. This can be found in Outputs tab for your new stack, with the key ```MATLABProductionServerApplicationsBucket```. 
+4. Select **Move**.  
+
+### Step 4. Verify Access to MATLAB Production Server Dashboard and Applications for New Deployment
+1. In the **Outputs** tab for your new stack, locate the ```MatlabProductionServerDashboardURL``` key and click the corresponding URL. This takes you to the dashboard login page.  
+2. Log in using the administrator username and password that you specified in the **Dashboard Login** section while creating the stack. 
+3. Confirm that the displayed server information matches the version and operating system specified during deployment, and that all deployed applications appear in the **Applications** tab. 
+
+### Step 5. Delete Unused Resources from Previous Deployment
```

Comment:
Starting in 23b, there is an additional CustomLambda stack that we can include in this list.